lib/generate/index.test.js

```diff
@@ -136,6 +136,27 @@ test('should not inject anything if end tag is malformed', t => {
   t.is(result, content);
 });
 
+test('should inject nothing if there are no contributors', t => {
+  const {options, content} = fixtures();
+  const contributorList = [];
+  const expected = [
+    '# project',
+    '',
+    'Description',
+    '',
+    '## Contributors',
+    'These people contributed to the project:',
+    '<!-- ALL-CONTRIBUTORS-LIST:START -->',
+    '<!-- ALL-CONTRIBUTORS-LIST:END -->',
+    '',
+    'Thanks a lot guys!'
```

Comment:
:+1: will change it right away!
Suspiciously timely with a discussion on some podcast...